metrics/core/src/test/java/org/hyperledger/besu/metrics/prometheus/MetricsHttpServiceTest.java

```diff
@@ -174,6 +174,22 @@ public void metricsAreAbsentWhenFiltered() throws Exception {
     }
   }
 
+  @Test
+  // There is only one available representation so content negotiation should not be used
+  public void acceptHeaderIgnored() throws Exception {
+    final Request metricsRequest =
+        new Request.Builder().addHeader("Accept", "text/xml").url(baseUrl + "/metrics").build();
+    try (final Response resp = client.newCall(metricsRequest).execute()) {
+      assertThat(resp.code()).isEqualTo(200);
+      // Check general format of result, it maps to java.util.Properties
+      final Properties props = new Properties();
+      props.load(resp.body().byteStream());
+
+      // We should have JVM metrics already loaded, verify a simple key.
+      assertThat(props).containsKey("jvm_threads_deadlocked");
+    }
```

Comment:
```suggestion

      // Assert we still get the Content-Type Prometheus likes
      assertThat(resp.header("Content-Type")).isEqualTo(TextFormat.CONTENT_TYPE_004);
    }
```

You will also need `import io.prometheus.client.exporter.common.TextFormat;` but GitHub won't let me suggest that far our of the code diff.

Reply:
Added.